### DISCUSSION TITLE
Typing in date/timepicker

### DISCUSSION THREAD

**Discussion by el-klo** [filamentphp/filament#6209] [Help]
The date, time & datetimepickers work great but i want to be able to type in the date and/or time instead of just selecting it with my mouse. Can't seem to find how to enable this. Can anyone point me in the right direction?

**Comment by el-klo** [comment]
Not implemented so used another datepicker

**Reply by mashrab** [reply]
could you share you solution?

**Comment by dennyvik** [comment]
did you try with native(false) or without?
